make.nsi

```diff
@@ -121,6 +123,18 @@ Section /o '' sec21x64
     !insertmacro execSection 21 2019 x64
   ${ENDIF}
 SectionEnd
+  
+  Section /o '' sec22
```

Comment:
Thanks for your contribution. But it would be great if you remove these extra spaces before `Section` and update version on 5 line `OutFile '${MyName} Installer X7-2019.exe'` => `X7-2020`